offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -134,6 +134,32 @@ class OfferToPDFConverter implements OfferExporter {
      */
     private Map<ProductGroups, List<ProductItem>> productItemsMap = [:]
 
+    /**
+     * Max number of characters before line breaks in a property column in the productItem table
+     *
+     * This enum stores the maximum number of characters before a line breaks occurs in product property column in the productItem table
+     */
+    enum ProductPropertySpacing {
+        PRODUCT_NAME(33),
+        PRODUCT_DESCRIPTION(62),
+        PRODUCT_UNIT(15),
+        PRODUCT_UNIT_PRICE(15),
+        PRODUCT_AMOUNT(8),
+        PRODUCT_TOTAL(15)
+
+
```

Comment:
nice!

Reply:
You could also use simple variables e.g private final int PRODUCT_NAME = 33, I would even prefer it over an enum since this would be more straight forward.

Reply:
Since i can then also define it in the relevant method this makes sense to me 👍